components/event_dispatcher/introduction.rst

```diff
@@ -502,84 +457,28 @@ event object. This means that all listeners have direct access to the
 object's :method:`Symfony\\Component\\EventDispatcher\\Event::getDispatcher`
 method.
 
-This can lead to some advanced applications of the ``EventDispatcher`` including
-letting listeners dispatch other events, event chaining or even lazy loading
-of more listeners into the dispatcher object. Examples follow:
+This can lead to some advanced applications of the ``EventDispatcher``
+including dispatching other events inside listeners, chaining events or even
+lazy loading listeners into the dispatcher object.
```

Comment:
Removed the example because of 2 reasons:

 1. Imo, there were to many examples in this section; it's talking about a tiny feature.
 2. I do not believe any of these "advanced applications" are good practices. I've discussed with some people and we cannot come up with a listener that has to trigger a new event. In any such cases, the first one had to be a command instead of an event.